tests/Handler/AddPrefixCommandHandlerTest.php

```diff
@@ -84,7 +84,11 @@ protected function tearDown()
     {
         chdir($this->workingDirectory);
         $filesystem = new Filesystem();
-        $filesystem->remove($this->tempDir);
+        if (!defined('PHP_WINDOWS_VERSION_BUILD')) {
+            $filesystem->remove($this->tempDir);
+        } else {
+            exec(sprintf("rd /s /q %s", escapeshellarg($this->tempDir)));
```

Comment:
why is this workaround needed? I would expect the filesystem to take care of such platform specific issues